### PR TITLE
Bump osb-broker-lib to 0.0.5

### PR DESCRIPTION
**Pull request by pmorie**

### CODE REVIEW

**Comment by LiliC** [comment]
We should be setting the `response.Exists` to `true` in the bind and provision, as this is also introduced in this bump of osb-lib version as well. See [here](https://github.com/pmorie/osb-broker-lib/blob/0.0.5/pkg/rest/apisurface.go#L111). Otherwise the broker tries to re-provision as the correct status is not returned. Or maybe document in the code at least, WDYT?

File: openshift/starter-pack.yaml [hunk 1]
```diff
@@ -54,6 +54,9 @@ objects:
           args:
           - --port
           - "8080"
+          - -v
+          - "4"
+          - --logtostderr
```
Comment (carolynvs) [inline openshift/starter-pack.yaml]: 👍

File: pkg/broker/logic.go [hunk 2]
```diff
@@ -59,41 +60,41 @@ services:
...
-              "default": "My Name"
-          - "color":
-              "title": "Color"
-              "type": "string"
-              "default": "Clear"
-              "enum":
-              - "Clear"
-              - "Beige"
-              - "Grey"
+          parameters:
```
Comment (carolynvs) [inline pkg/broker/logic.go]: This is so much nicer! 🎉

File: pkg/broker/logic.go [hunk 3]
```diff
@@ -36,64 +36,73 @@ type BusinessLogic struct {
...
-		return nil, err
+	response := &broker.CatalogResponse{}
+	osbResponse := &osb.CatalogResponse{
+		Services: []osb.Service{
+			{
+				Name:          "example-starter-pack-service",
+				ID:            "4f6e6cf6-ffdd-425f-a2c7-3c9258ad246a",
+				Description:   "The example service from the osb starter pack!",
+				Bindable:      true,
+				PlanUpdatable: truePtr(),
```
Comment (LiliC) [inline pkg/broker/logic.go]: 👍 :D

File: pkg/broker/logic.go [hunk 4]
```diff
@@ -36,64 +36,73 @@ type BusinessLogic struct {
...
+								},
+							},
+						},
+					},
+				},
+			},
+		},
 	}
 
+	glog.Infof("catalog response: %#+v", osbResponse)
```
Comment (LiliC) [inline pkg/broker/logic.go]: I think we can get rid of this?

**Pull request merged**